src/jwtcookie/Authentication/JwtCookie/JwtCookieHandler.cs

```diff
@@ -179,5 +216,23 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
                 throw;
             }
         }
+
+        /// <summary>
+        /// Get the signing keys published by the given endpoint.
+        /// </summary>
+        /// <param name="url">The url of the endpoint</param>
+        /// <returns>The signing keys published by the endpoint</returns>
+        public async Task<ICollection<SecurityKey>> GetSigningKeys(string url)
```

Comment:
@SandGrainOne du drev vel litt å se på dette. Hvordan er det med caching her. Vil det være en innebygget caching her som gjør at man ikke henter signings key for samme hver gang?

Reply:
Jeg mener det er innebygget caching her et sted. Fant dette dere kan nøyste videre i:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.tokens.baseconfigurationmanager.refreshinterval?view=azure-dotnet#microsoft-identitymodel-tokens-baseconfigurationmanager-refreshinterval